### PR TITLE
Help Content

### PR DESCRIPTION
**Pull request by luisgov**
Begin adding content for the help document:
https://docs.google.com/document/d/1Y6RbHd0YMwDpxxowP07MvJvkfvxhsAgmKiYNjoVm9bo/edit#/

Content is not final. Some work still needs to get done to handle overflowing tables.

screencapture-localhost-8000-help-2018-09-25-16_29_33

### CODE REVIEW

File: templates/help/index.html [hunk 1]
```diff
@@ -40,15 +40,148 @@
...
+          <h2 id='financial-verification'>Financial Verification</h2>
+
+          <h3 id='how-to-prepare-for-financial-verification-step'>How to prepare for Financial Verification step?</h3>
+          <p>Once your request is approved, the next step is to create a Task Order (T.O.) associated with the JEDI Cloud ID/IQ.  Please contact a Contracting Officer (KO) or Contracting Officer Representative (COR) to help with this step. </p>
+          <p>This may also involve talking to your Financial Manager (FM) to secure funding.</p>
+          <p>Once the Task Order (T.O.) has been created, you will need to provide information related to the task order and funding in AT-AT. This step is referred to as “Financial Verification.”</p>
+          <p><em>We also recommend getting familiar with the <a href="#">JEDI Cloud CLIN structures</a> so that you know which specific services are available under JEDI and categorized for contracting purposes. This will help you and the Contracting Officer create a Task Order.</em></p>
```
Comment (patricksmithdds) [inline templates/help/index.html]: Should this link be to `#how-are-the-jedi-idiq-clins-structured`?

**Comment by andrewdds** [comment]
Can we put some manual column widths (totaling 100%) on those tables, so they don't extend off the page?

File: templates/help/index.html (hunk 1 above)
Comment (luisgov) [reply templates/help/index.html]: 👍

**Comment by andrewdds** [comment]
@luisgov I added some changes to this PR.

I made the helpdocs route do routing to a real filename in `help/docs`, and generate a navigation based on these files.

The `help/index` is just a base page to the help docs, and will eventually need some other content and design

Also added some styles for fixed width tables

**Pull request merged**